WeatherPlannerDemo/Utils/AirPollutionViewModel+Extension.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+extension AirPollutionViewModel {
+    func airPollutionDescription(_ airPollutionIndex: Int) -> String? {
+        switch airPollutionIndex {
+        case 1:
+            return LocalizationKey.airQualityGood.string
+        case 2:
+            return LocalizationKey.airQualityFair.string
+        case 3:
+            return LocalizationKey.airQualityModerate.string
+        case 4:
+            return LocalizationKey.airQualityPoor.string
+        case 5:
+            return LocalizationKey.airQualityVeryPoor.string
+        default:
+            return nil
+        }
+    }
+    
+    func airPollutionData() -> [(name: String, mark: String, value: String)] {
```

Comment:
You can create AirPoullutionDetailsViewModel with name, mark and value properties. This is more readable than tuple.
Also, this can be lazy computed property:
```
lazy var airPolutionDetails: AirPoullutionDetailsViewModel = {
   ...
}()
```